arviz/stats/stats.py

```diff
@@ -1045,6 +1040,11 @@ def summary(
         Dimensions specification for the variables to be used if the ``fmt`` is ``'xarray'``.
     credible_interval: float, optional
         deprecated: Please see hdi_prob
+    order
+        deprecated: order is now ignored.
```

Comment:
Why is order removed?

Reply:
or is there a way to select order for unwrapped values?

Reply:
I think that having informative labels, order is not really necessary (also it is not present in any plot and nobody has complained). High dimension and confusing cases should probably better use `dim: coord_value` format or something like this. 

One of the goals here is to make summary and plots use the same labeling and between removing order from summary or adding order to plots I went for removing order from summary.

Reply:
Weren't we talking about returning xarray's of axes (or other backend entities) as a way to do this?  TBH, I don't know if it's possible to move around the subplots in matplotlib -- if this can be done, then the xarray solution would work.